package.json

```diff
@@ -16,10 +16,10 @@
     "url": "https://github.com/M6Web/superagent-mock"
   },
   "engines": {
-    "node": ">=10"
+    "node": ">=16.0.0"
   },
   "peerDependencies": {
-    "superagent": ">=3.6.0"
+    "superagent": ">=8.0.9"
```

Comment:
Why is superagent a peer dependency instead of direct dependency ?

Reply:
`superagent-mock` is not in responsibility for installing `superagent` in the project where it is used